compiler/src/dmd/astenums.d

```diff
@@ -63,7 +63,7 @@ enum STC : ulong  // transfer changes to declaration.h
     foreach_            = 0x4000,   /// variable for foreach loop
     variadic            = 0x8000,   /// the `variadic` parameter in: T foo(T a, U b, V variadic...)
 
-    //                  = 0x1_0000,
+    inscoperef          = 0x1_0000,   /// when `in` means in|scope|ref
```

Comment:
```suggestion
    inscoperef          = 0x1_0000,   /// when `in` means const|scope|ref
```

Reply:
I also changed `inscoperef` to `constscoperef`